app.rb

```diff
@@ -1,9 +1,26 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/peeps' do
+    @message = Peep.all
+
+    erb :peeps
+  end
+
+  get '/peeps/new' do
+    erb :"peeps/new"
+  end
+
+  post '/peeps' do
+    Peep.create(message: params[:message])
+
+    redirect 'peeps'
+  end
+
```

Comment:
You have used 'post' to handle the data then redirect to get, which is the correct way of doing it. I'm guilty of handling all in the post request.